routes/orderRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+const orderController = require('../controllers/orderController');
+const { loginRequired } = require('../utils/auth');
+
+router.get('/searching', loginRequired, orderController.getOrderByOrderId);
+router.get('/searchingAll', loginRequired, orderController.getAllOrderByUserId);
```

Comment:
이미 router 모듈에서 `/order`의 path가 설정되어 있기 때문에 해당 엔드포인트를 명시하지 않아도 될 것 같습니다!
```suggestion
router.get('/', loginRequired, orderController.getAllOrderByUserId);
```